driver/modern_bpf/helpers/extract/extract_from_kernel.h

```diff
@@ -64,6 +64,39 @@ static __always_inline u32 extract__syscall_id(struct pt_regs *regs)
 static __always_inline unsigned long extract__syscall_argument(struct pt_regs *regs, int idx)
 {
 	unsigned long arg;
+#if defined(__TARGET_ARCH_x86)
+	// TODO: somehow use syscalls_dispatcher__check_32bit_syscalls()
```

Comment:
Didn't want to move `syscalls_dispatcher__check_32bit_syscalls` to `extract_from_kernel`.
@Andreagit97 wdyt?